<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>848</width>
    <height>711</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Simulation</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="MainSceneGL" name="openGLWidget">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>800</width>
      <height>600</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>620</y>
      <width>651</width>
      <height>31</height>
     </rect>
    </property>
    <property name="maximum">
     <number>10000</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="format">
     <string/>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>610</y>
      <width>86</width>
      <height>53</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="startScene">
       <property name="text">
        <string>Start Scene</string>
       </property>
       <property name="icon">
        <iconset>
         <normalon>:/new/prefix1/res/images/play-button_318-42541.png</normalon>
        </iconset>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButton">
       <property name="text">
        <string>Stop Scene</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>848</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MainSceneGL</class>
   <extends>QOpenGLWidget</extends>
   <header>MainSceneGL.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
